README.md

```diff
@@ -45,6 +48,11 @@ to learn about basic usage.
 Learn more at our [documentation](https://detectron2.readthedocs.org).
 And see [projects/](projects/) for some projects that are built on top of detectron2.
 
+DensePose pre-trained models have been made available in the form of hosted APIs by third parties 
```

Comment:
`DensePose pre-trained models demo:`

"third parties", "inference testing" create opportunities for scrutiny from reviewers IMO